lib/lobby.ts

```diff
@@ -0,0 +1,160 @@
+import { getClient } from '.';
+import { DbItem, IDbItem } from './db-item';
+import { COLLECTION } from './private/enums';
+
+type DatabaseEntry = Omit<ILobby, 'collectionName' | 'existsInDb'>;
+type ClientResponse = Omit<DatabaseEntry, 'users' | 'uri'>;
+
+export interface LobbyProps {
+  /**
+   * The id of the lobby creator
+   */
+  readonly ownerId: string;
+
+  /**
+   * A lobby's users
+   */
+  readonly users: string[];
+
+  /**
+  * The spotify playlist id
+  */
+  readonly playlistId?: string;
+
+  /**
+  * The theme for the lobby
+  */
+  readonly theme: string;
+
+  /**
+  * The name of the lobby
+  */
+  readonly name: string;
+}
+
+export interface ILobby extends Omit<LobbyProps, 'topSongs'>, IDbItem {
+}
+
+/**
+ * The class containing a user and their data
+ */
+export class Lobby extends DbItem implements ILobby {
+  /**
+   * A static function to query for a user from their id
+   *
+   * @returns a user object if the id exists in the database
+   */
+  public static async fromId(id: string): Promise<Lobby | null> {
+    const client = await getClient();
+    const document = await client.findDbItem(COLLECTION.LOBBIES, id);
+    if (!document) return null;
+    const content: DatabaseEntry = document.getContent() as DatabaseEntry;
+    return new Lobby(id, content, true);
```

Comment:
change this to the `key` of the document! makes it easy to replace!

Reply:
wait change which part to the `key`?

Reply:
https://github.com/UCLA-Creative-Labs/aurgy-backend/blob/55b60400d4c6073306cb3b2f01d97d9b4acb4597/lib/user.ts#L64-L70